backend/internal/model/game.go

```diff
@@ -145,3 +154,34 @@ func (g *game) GetShotsData() ShotsData {
 
 	return g.shotsData
 }
+
+func (g *game) UpdateManualGoals(teamID int, action string) error {
+	g.m.Lock()
+	defer g.m.Unlock()
+
+	switch action {
+	case "add":
```

Comment:
I found this string is used in the same project [here](https://github.com/HackYourCareer/SmartKickers/pull/100/files#diff-cdf83dc7e0d670770a64561f00209cdedc6691efdf728c34a3e1ba8fbf54e6b8R146) - I would suggest moving it and the sub to the one place (for example as const value) to  add a little context to it and make it easy to maintain